src/index.ts

```diff
@@ -1,8 +1,9 @@
-import app from './app';
-import config from './config';
+import App from '@app';
+import envVar from '@validations/envVar';
+import { dbConnection } from '@database/index';
 
-const { port } = config.server;
+envVar();
 
-app.listen(port, () => {
-  console.log('Server is running on port:', port);
-});
+const app = new App([], dbConnection);
```

Comment:
The empty array should be removed due to the requested change in the `App` class constructor comment,

Reply:
We have to pass the router constructors, not complete yet
Thanks